src/style.css

```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: oblique;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 16px;
+}
+
+.search {
+  display: flex;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  line-height: 22px;
+}
+
+.big .big-icon {
+  width: 19px;
+  margin: 25px 17px 26px 26px;
+}
+
+.big .big-input {
+  all: unset;
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
```

Comment:
but if I make it 900 on focus, it will impact a placeholder also, so on hocus the placeholder becomes 900 and on blur goes back to 300, and we want only text inside the input to be impacted by this styles. So maybe I don't understand completely how it works